packages/date/tests/utils.test.js

```diff
@@ -105,7 +105,7 @@ describe('utils', () => {
     it('renders correct options when min and max are strings', () => {
       const month = moment();
       const min = '01/01/1999';
-      const max = '12/31/2020';
+      const max = '12/31/2021';
```

Comment:
Looking forward to seeing this change again next year 🤣 Since we are here would you mind changing this to use `moment` and whatever the current year is? If not it's not a blocker, I can go change it

Reply:
Lol I thought about that when I changed but didn't want to change it from the string format. Guess we can use moment but keep it as a string.